spec/features/collection_drafts/forms/required_fields_spec.rb

```diff
@@ -153,7 +153,7 @@
       end
 
       it 'displays the required icons' do
-        expect(page).to have_css('label.eui-required-o', count: 24)
+        expect(page).to have_css('label.eui-required-o', count: 29)
```

Comment:
Since you're making a change here, it might be nice to document what the 29 expected required icons are coming from (generally) in a comment.